### PR TITLE
Add profile chinmay29hub

### PR DESCRIPTION
**Pull request by chinmay29hub**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1129"></a>

### CODE REVIEW

**Review by kabir0x17 — APPROVED** [review]
Looks good to me 👍
Keep contributing to open-source

**Review by KendallDoesCoding — APPROVED** [review]

**Review by syedsohan — APPROVED** [review]

File: public/data/chimay29hub.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/chinmay29hub",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/chinmay-sonawane-38474019a/",
```
Comment (KendallDoesCoding) [inline public/data/chimay29hub.json:19]:
```suggestion
      "url": "https://www.linkedin.com/in/chinmay-sonawane-38474019a",
```

The / is not needed in the url, it works anyway. It just gets added when you copy the url from the taskbar.

**Review by KendallDoesCoding — APPROVED** [review]

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]
The / is not needed in the url, it works anyway. It just gets added when you copy the url from the taskbar.

**Comment by KendallDoesCoding** [comment]
Sorry for duplicate reviews, I clicked approve instead of suggest changes.

File: public/data/chimay29hub.json (hunk 1 above)
Comment (Panquesito7) [reply public/data/chimay29hub.json:19]: There's no harm in keeping the `/` here. I suggest we keep it as-is.

**Review by Panquesito7 — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
LGTM!, thank you for the contribution :rocket:

**Comment by qwel-exe** [comment]
LGTM :rocket: thanks for your contributions we will merg this soon :)

**Pull request merged**